src/client/components/pages/parts/revisions-table.js

```diff
@@ -116,11 +116,19 @@ function RevisionsTable(props) {
 														revision.notes.map(note => (
 															<div className="revision-note clearfix" key={note.id}>
 																<span className="note-content">
-																	{note.content}
+																	<span
+																		dangerouslySetInnerHTML={{
```

Comment:
hey @MonkeyDo  is it fine here to use `dangerouslySetInnerHTML` , if not can you suggest any alternate option we can try.

Reply:
I like to add the package `DOMPurify` whenever I use dangerouslySetInnerHTML , just to be on the safe side .

Here a good breakdown of how it's used: https://linguinecode.com/post/complete-guide-react-dangerouslysetinnerhtml

Reply:
thank you, will make the changes soon :)